src/Sylius/Bundle/ApiBundle/ApiPlatform/ApiResourceConfigurationMerger.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\ApiPlatform;
+
+/** @experimental */
+class ApiResourceConfigurationMerger implements ApiResourceConfigurationMergerInterface
```

Comment:
```suggestion
final class ApiResourceConfigurationMerger implements ApiResourceConfigurationMergerInterface
```

Reply:
im unable to write tests for it when its final (same with ResourceMetadataPropertyValueResolver)

Reply:
You should use interface in specs